Source/WebCore/editing/cocoa/AttributedString.mm

```diff
@@ -383,7 +383,7 @@ static MultiRepresentationHEICAttachmentData toMultiRepresentationHEICAttachment
     }, [] (const RetainPtr<NSDate>& value) -> RetainPtr<id> {
         return value;
     }, [] (const Ref<Font>& font) -> RetainPtr<id> {
-        return (__bridge PlatformFont *)(font->getCTFont());
+        return (__bridge PlatformFont *)(font->ctFont());
```

Comment:
I don't think the extra parentheses that were here are helpful and we can remove them:
```suggestion
        return (__bridge PlatformFont *)font->ctFont();
```